.codeclimate.yml

```diff
@@ -24,25 +21,24 @@ engines:
       - javascript
   eslint:
     enabled: false
-    channel: "eslint-3"
+    channel: eslint-3
   fixme:
-    # let's enable later
     enabled: false
   markdownlint:
-    # let's enable later
     enabled: false
-  rubocop:
-    enabled: true
-    config: '.rubocop_cc.yml'
-    channel: rubocop-0-69
 prepare:
   fetch:
-  - url: "https://raw.githubusercontent.com/ManageIQ/guides/master/.rubocop_base.yml"
+  - url: https://raw.githubusercontent.com/ManageIQ/manageiq-style/master/.rubocop_base.yml
     path: ".rubocop_base.yml"
-  - url: "https://raw.githubusercontent.com/ManageIQ/guides/master/.rubocop_cc_base.yml"
+  - url: https://raw.githubusercontent.com/ManageIQ/manageiq-style/master/.rubocop_cc_base.yml
```

Comment:
Topological Inventory switched to Insights rucobop files. I'm wondering whether we can set different urls for master and 0.2.z branches

Reply:
That feels a little bit strange to me.  I think it depends on which project (ManageIQ or Topo) "owns" this gem going forward.  Since it currently lives in the ManageIQ namespace I'm leaning toward ManageIQ.

Reply:
I'd rather not have different rules for master and older branches.  This is a released gem and the style it uses shouldn't impact the projects that use it.  My goal is still to have MIQ use inventory_refresh v0.3 once we reconcile some of the incompatible changes that were made for topological inventory that aren't going to work for manageiq.

Reply:
And as @bdunne points out this is in the ManageIQ org so it makes sense to follow the manageiq style rules